src/common/runner/runner.go

```diff
@@ -173,6 +184,9 @@ func (r *Runner) TagFile(file string) {
 			if r.isSkippedResourceType(block.GetResourceType()) {
 				continue
 			}
+			if r.isSkippedResource(block.GetResourceID()) {
```

Comment:
Since `block.GetResourceID()` is implemented differently for each framework, I would document the different options to skip in the `DefaultText` in `main.go`

Reply:
sure i will try to improve the docs